hack/jenkins/e2e.sh

```diff
@@ -104,8 +104,22 @@ else
 
     sudo flock -x -n /var/run/lock/gcloud-components.lock -c "gcloud components update -q" || true
 
-    GITHASH=$(gsutil cat gs://kubernetes-release/ci/latest.txt)
-    gsutil -m cp gs://kubernetes-release/ci/${GITHASH}/kubernetes.tar.gz gs://kubernetes-release/ci/${GITHASH}/kubernetes-test.tar.gz .
+    # VERSION_FILE defaults to "latest", but can also be set to "stable" for
+    # official releases.
+    VERSION_FILE="latest"
```

Comment:
Local variables can only be used within a function. This isn't in a function, right?